test/runtests.jl

```diff
@@ -459,4 +459,31 @@ using Test
         @test length(net.buses) == 2
         @test length(net.loads) == 1
     end
+
+    @testset "empty network" begin
+
+        empty_net = Network(
```

Comment:
Slightly long winded test set up.  Is there perhaps a tidier constructor for this?

Reply:
yeah... i think it's okay

I suppose an alternative would be to define `Network()` as a way to create an empty `Network`
and/or implement `empty!(x::Network)`

But i'm fine with this as it is